tortoise_interpreter.py

```diff
@@ -1,51 +1,77 @@
+"""Implementation of tortoise-language."""
+
+
 def main_tortoise(program, print_to_screen=print):
+    """Main function that interprets a tortoise program."""
     program = sanitize_program(program)
-    COMMANDS = {'U': pen_up, 'D': pen_down, 'P': pen_colour, 'N': move_north, 'S': move_south, 'E': move_east, 'W': move_west}
-    command_to_parse = navigate_command_functions(COMMANDS, program)
+    commands = {
+        "U": pen_up,
+        "D": pen_down,
+        "P": pen_colour,
+        "N": move_north,
+        "S": move_south,
+        "E": move_east,
+        "W": move_west,
+    }
+    command_to_parse = navigate_command_functions(commands, program)
 
     for command in command_to_parse:
         print_to_screen(command)
 
 
 def sanitize_program(program):
+    """Get tortoise program ready for interpretation."""
```

Comment:
This docstring is a little vague; what makes it "ready for interpretation"? Maybe combine this one-line docstring with the larger comment below, and explain why this is required?